cea/technologies/thermal_network/thermal_network_matrix.py

```diff
@@ -103,10 +104,16 @@ def thermal_network_main(locator, gv, network_type, source, set_diameter):
 
     # # prepare data for calculation
 
-    # read building names
+    # read building names from the entire district
     total_demand = pd.read_csv(locator.get_total_demand())
     building_names = total_demand['Name']
```

Comment:
this line can be replaced by `building_names = pd.read_csv(locator.get_total_demand())['Name'].values`. This prevents loading the entire dataframe

Reply:
done!